includes/definitions/sti410c.yaml

```diff
@@ -0,0 +1,10 @@
+os: sti410c
```

Comment:
Is this OS name too specific to one model? Ideally an OS covers a range of models in a product line.